liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/applications/LooseEarApplication.java

```diff
@@ -121,7 +128,7 @@ public String getModuleUri(MavenProject proj) throws Exception {
                                 if (bundleFileName != null) {
                                     return bundleDir + "/" + bundleFileName;
                                 } else {
-                                    return bundleDir + "/" + getModuleName(proj.getGroupId(), proj.getArtifactId(), proj.getVersion(), proj.getPackaging());
+                                    return bundleDir + "/" + getModuleName(groupId, artifactId, version, type);
```

Comment:
@dblitza Heads up. For the fix you are working on, you will need to merge with this PR.